app/assets/stylesheets/nucore.scss

```diff
@@ -628,3 +634,44 @@ li:hover {
 .nav.pull-right {
   letter-spacing: $fontSpacing;
 }
+
+
+/*
+ UCONN Redesign Table View
+*/
+td, tr, {
+  transition: ease 0.2s all;
+  border-bottom: solid  $grey 1px;
+}
+.table-striped tbody > tr:nth-child(2n+1) {
+  td {
+    background-color: $lighterGrey;
+  }
+  th {
+    background-color: $lighterGrey;
+  }
+} 
+
+.table {
+  td {
+    background-color: $headerBackgroundColor;
```

Comment:
I don't think this is a good option. I think it should probably be one of the grays. In the other schools we override this variable. In NU it's purple and Dartmouth it's green.